src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -102,6 +104,7 @@ default String getAsTextHidePrivate() {
                 getEmail(),
                 getAddress(),
                 getFees());
+        getAttendance();
         builder.append(stringChain)
```

Comment:
You want to show the attendance when we list people? Wouldn't that extend the list quite a lot?